src/content/community/rollup-grants/index.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Ethereum Rollup Community Grants
```

Comment:
If this is focussed on Optimistic rollups should this be reflected in the title?

Reply:
We had some discussions and want to pivot on this. I changed the language to cover all rollups and mostly removed the "optimistic" part in headers.